lessons/lesson5/lesson.md

```diff
@@ -0,0 +1,124 @@
+### Writing docstrings
+
+We'll continue with the webserver library from the exercise.
+
+%(example1/example1.jsonnet)s
+
+This library provides a number of functions to create a webserver. In docsonnet lingo this
```

Comment:
Add a link to docsonnet here?

Reply:
The intro for this page is in `main.jsonnet` which has a link.